WBooks/Library/LibraryViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  LibraryViewController.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 8/21/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class LibraryViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    struct Constants {
+        static let backgroundBlueColor = UIColor(hex: "EAF6FA")
```

Comment:
If this color will be used in many places of the app, you can create a global color constants (sometimes we do it by an extension in UIImage) with all the colors of the app (usually these are things we get in the design style guides that define for example the app's palette)